src/hooks/chats/venue/useCanDeleteVenueChatMessages.tsx

```diff
@@ -1,16 +1,19 @@
+import { World } from "api/world";
+
 import { AnyVenue } from "types/venues";
 
-import { useRoles } from "hooks/useRoles";
 import { useUser } from "hooks/useUser";
 
-export const useCanDeleteVenueChatMessages = (venue: AnyVenue) => {
+export const useCanDeleteVenueChatMessages = (
+  venue: AnyVenue,
+  world?: World
+) => {
```

Comment:
instead of positional argument, a single options object allows for forward and backward compatibility in the future

Reply:
Fixed in https://github.com/sparkletown/sparkle/pull/2820/commits/92eecf9525feec32828635f3869524ca12e85a99